js/deer-config.js

```diff
@@ -589,7 +594,12 @@ export default {
                         return
                     }
                     queryString = queryString.trim()
-                    const query = decodeContentState(queryString)
+                    const query = decodeContentState(queryString);
+                    for (const key in query) {
```

Comment:
We don't want to be altering the original query data. Something else may need it to be case sensitive. Do not change query.